content/en/account_management/audit_trail/events.md

```diff
@@ -101,9 +101,13 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | [User login][29]                     | A user logs into Datadog and the authentication method used.                                  | `@evt.name:Authentication @action:login`                |
 
 ### CI Visibility events
-| Name                            | Description of audit event                                                                             | Query in audit explorer                                                                          |
-|---------------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| [Repository Default Branch][30] | A user modified the default branch of a repository, and the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`   
+| Name                            | Description of audit event                                                                                  | Query in audit explorer                                              |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
+| [Repository Default Branch][30] | A user modified the default branch of a repository, and the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`            |
```

Comment:
```suggestion
| [Repository Default Branch][30] | A user modified the default branch of a repository, as well as the previous and new values for the default branch. | `@evt.name:"CI Visibility" @asset.type:ci_app_repository`            |
```

Reply:
Hi Rosa, I'm actually not changing this description in this PR. But it seems strange, given that the user is not modifying 'the previous and new values for the default branch', it's just updating the default branch. In any case, the previous and the new values are shown as part of the audit event. I may rephrase it as:

> A user modified the default branch of a repository.

What do you think?

Reply:
That works and seems more clear/succinct! :)

Reply:
Great, I've pushed it :)